liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -194,6 +197,16 @@ protected void runMojo(String groupId, String artifactId, String goal) throws Mo
         executeMojo(plugin, goal(goal), config,
                 executionEnvironment(project, session, pluginManager));
     }
+    
+    protected void runExplodedMojo() throws MojoExecutionException {
+        Plugin warPlugin = getPlugin("org.apache.maven.plugins", "maven-war-plugin");
```

Comment:
Same question about `ear` applications. What path do they take?

Reply:
If `runExplodedMojo` is only called for `war` applications, that should be documented here. This logic looks like it would only apply to a `war` app.